pkg/vcs/github_client/message.go

```diff
@@ -16,7 +16,75 @@ import (
 	"github.com/zapier/kubechecks/telemetry"
 )
 
-const MaxCommentLength = 64 * 1024
+const (
+	MaxCommentLength = 64 * 1024
+
+	// Using string concatenation for better code readability.
+	// Cannot use raw string literal (backticks) here as the separators themselves contain backticks.
+	sepStart = "Continued from previous comment.\n" +
```

Comment:
Could you add the identifier to your message continued message? Here's a breakdown of the change https://github.com/zapier/kubechecks/pull/370